server/src/test/java/org/apache/druid/server/http/CoordinatorDynamicConfigTest.java

```diff
@@ -89,6 +89,7 @@ public void testSerde() throws Exception
         9,
         false,
         false,
+        Integer.MAX_VALUE,
```

Comment:
There are a pile of these. Should we introduce a builder or some other way to reduce redundancy?